daft/session.py

```diff
@@ -35,6 +36,97 @@ def _from_env() -> Session:
         # todo session builders, raise if DAFT_SESSION=0
         return Session.empty()
 
+    ###
+    # exec
+    ###
+
+    def exec(self, input: str) -> DataFrame:
+        return self._session.exec(input)
```

Comment:
Is this the equivalent of `daft.sql(...)`? We should perhaps add a little docstring here too